src/chains/index.ts

```
@@ -0,0 +1,21 @@
+import * as viemChains from 'viem/chains';
+import { arthera } from './arthera';
+import { fiveIre } from './fiveIre';
+import { hyperEvm } from './hyperEvm';
+import { hyperliquid } from './hyperliquid';
```

ESLint

[error] Missing file extension for "./hyperliquid"

(import/extensions)